### DISCUSSION TITLE
Nunjucks still appearing as HTML+Django in language list

### DISCUSSION THREAD

**Discussion by Nixinova** [github-linguist/linguist#5220] [Classification] [answered]
Nunjucks was split from HTML+Django in #5167, and the language bar now shows Nunjucks separately ([proof](https://github.com/stoph/eleventy-base-blog)), but all files are still classified as "HTML+Django" in [search](https://github.com/search?q=extension%3Anjk) etc, even those indexed after 7.13 was released.

Is this an issue with Linguist or just prior caching?

**Comment by Nixinova** [comment]
Sidenote, you may want to rename the category name... Maybe "Language Identification"?
![image](https://user-images.githubusercontent.com/42429413/108306509-2b377f80-7211-11eb-9c33-c9dfa42e269f.png)

**Reply by Alhadis** [reply]
LOL.

@lildude Jarring cutoff-points aside, the category's label is too damn long. I recommend  _"Classification"_ instead.

**Reply by lildude** [reply]
I'm a little scared "Classification" may not be clear enough for peeps who don't use English as their primary language, but I've gone ahead and renamed it to "Classification" anyway. I can change this if we start to see discussions popping up in the wrong category or unnecessary issues.

**Reply by Alhadis** [reply]
> I'm a little scared "Classification" may not be clear enough for peeps who don't use English as their primary language

The same argument can be made for anything Western cultures consider "self-explanatory". Depending on language and cultural barriers, different readers will receive different undertones from diction that isn't so well-established as in the West. 

Hell, you're south of an entire _continent_ of cultures where assumptions about English go out the window.  I was amazed when I learned Ethopia has _five_ national languages, none of which are English—or even based on the Latin alphabet.

Sorry, I'm rambling. Too much time spent on [Wikipedia](https://en.m.wikipedia.org/wiki/Languages_of_Ethiopia)… Which, BTW, was the reason I slacked off on writing that grammar I promised this morning. 😅 I'll do it again, I promise…

**Comment by lildude** [comment, marked as answer]
> Is this an issue with Linguist or just prior caching?

It definitely not Linguist else the language bar wouldn't be correct. This is something going awry with the search functionality which is completely independent of Linguist. I'm starting to suspect it uses a different list of languages or even an older version of Linguist as we saw this in https://github.com/github/linguist/issues/5052 too.

I've opened an issue asking for our Search team to look into this.
